.github/workflows/compliance.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         # debug
         ls -la
         git log  --pretty=oneline | head -n 10
-        ./scripts/ci/check_compliance.py -m Codeowners -m Devicetree -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m Kconfig -c origin/${BASE_REF}..
+        ./scripts/ci/check_compliance.py -m Codeowners -m Devicetree -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m KconfigBasic -c origin/${BASE_REF}..
```

Comment:
Why is `KconfigBasic` needed here ?

When we run compliance in `sdk-zephyr`, then we use the `sdk-zephyr/west.yml` and thus have all needed repos available for running full Kconfig check, which should pass without errors.

When running compliance in `sdk-nrf` repo, we already uses `KconfigBasic` here:
https://github.com/nrfconnect/sdk-nrf/blob/4680001fd63b066eb688af3909bc31bbde99d7cf/.github/workflows/compliance.yml#L60
```
        $ZEPHYR_BASE/scripts/ci/check_compliance.py -m Codeowners -m Devicetree -m Gitlint -m Identity -m Nits -m pylint -m checkpatch -m KconfigBasic -c origin/${BASE_REF}..
```

Reply:
Because we have references to `CONFIG_` symbols in the code that are only present in sdk-nrf. IIRC it's the partition manager.

Reply:
right, had not taken into consideration that we have `[noup]` patches that would impact.
Thanks for clarifying.